src/lang/modifyAst/boolean.ts

```diff
@@ -1,331 +1,182 @@
 import type { Node } from '@rust/kcl-lib/bindings/Node'
 
-import type EditorManager from '@src/editor/manager'
-import type { KclManager } from '@src/lang/KclSingleton'
-import type CodeManager from '@src/lang/codeManager'
 import {
-  createArrayExpression,
   createCallExpressionStdLibKw,
   createLabeledArg,
-  createLocalName,
-  createVariableDeclaration,
-  findUniqueName,
 } from '@src/lang/create'
-import { updateModelingState } from '@src/lang/modelingWorkflows'
-import { getNodeFromPath } from '@src/lang/queryAst'
-import { getNodePathFromSourceRange } from '@src/lang/queryAstNodePathUtils'
-import { getFaceCodeRef } from '@src/lang/std/artifactGraph'
-import type { EngineCommandManager } from '@src/lang/std/engineConnection'
-import type {
-  Artifact,
-  ArtifactGraph,
-  Program,
-  VariableDeclaration,
-} from '@src/lang/wasm'
-import { EXECUTION_TYPE_REAL } from '@src/lib/constants'
+import { getVariableExprsFromSelection } from '@src/lang/queryAst'
+import type { ArtifactGraph, PathToNode, Program } from '@src/lang/wasm'
 import type { Selections } from '@src/lib/selections'
 import { err } from '@src/lib/trap'
-import { isArray } from '@src/lib/utils'
+import { KCL_DEFAULT_CONSTANT_PREFIXES } from '@src/lib/constants'
+import {
+  createVariableExpressionsArray,
+  setCallInAst,
+} from '@src/lang/modifyAst'
 
-export async function applySubtractFromTargetOperatorSelections(
-  solids: Selections,
-  tools: Selections,
-  dependencies: {
-    kclManager: KclManager
-    engineCommandManager: EngineCommandManager
-    codeManager: CodeManager
-    editorManager: EditorManager
-  }
-): Promise<Error | void> {
-  const ast = dependencies.kclManager.ast
-  const lastSolidsVars = getLastVariableDeclarationsFromSelections(
+export function addUnion({
```

Comment:
Here just renaming like sweeps and transforms with the same ins and outs. No unit testing here because we still have one pw test per codemod, but as soon as we wire edit flows (not needed yet but I did put in the nodeToEdit arg) or add new unlabeled args, we should move tests to a `boolean.test.ts` file